test/functional/http/get/delegates.js

```diff
@@ -313,7 +313,7 @@ describe('GET /delegates', () => {
 					.then(res => {
 						expect(res.body.data).to.have.length(13);
 						res.body.data.map(d => {
-							expect(/^genesis_1.*/.test(d.username)).to.be.true;
+							return expect(/^genesis_1.*/.test(d.username)).to.be.true;
```

Comment:
you can do the same here `res.body.data.map(d => expect(/99/.test(d.username)).to.be.true);`